### PR TITLE
Openrouter.ai support

### PR DESCRIPTION
**Pull request by zakhar-kogan**
# Changes

- [x] Added Openrouter, again as per #53
- [x] Added `litellm_uuid.txt` to `.gitignore` ⇢ should I? Looks like it should be personal to the developer/tester?

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,3 @@
 .venv
-.env
\ No newline at end of file
+.env
+litellm_uuid.txt
```
Comment (ishaan-jaff) [inline .gitignore:3]: can you undo this change please

Comment (ishaan-jaff) [reply .gitignore:3]: @krrishdholakia is this change ok ?

**Review by ishaan-jaff — COMMENTED** [review]
looks good! can you undo this gitignore change

**Comment by ghost** [comment]
cc: @zakhar-kogan

File: .gitignore (hunk 1 above)
Comment (ghost) [reply .gitignore:3]: that's fine - also a smart idea to not automatically push your uuid into the main repo

Comment (ghost) [reply .gitignore:3]: if that was the only blocker then this looks good to merge @ishaan-jaff

Comment (zakhar-kogan) [reply .gitignore:3]: Thanks for reviewing! <3 
Yeah, I was thinking the same - at best it would be replaced on running tests, but has a propensity to just mix things up.

Do I get it right that's OK now, no changes required?
Can revert in a whim if needed 🤔

**Comment by ghost** [comment]
merged to main
